examples/ionoscloud/compute/user.yaml

```diff
@@ -15,11 +15,11 @@ kind: User
 metadata:
   name: xplane-test-user
 spec:
-  3managementPolicies:
+  managementPolicies:
     - "*"
   forProvider:
     administrator: false
-    email: "unique@xplane.com"
+    email: "unique_10022@xplane.com"
```

Comment:
let's not give an actual email here, just in case someone actually wants to run this

Reply:
changed to a placeholder value ([e1497a0](https://github.com/ionos-cloud/crossplane-provider-ionoscloud/pull/140/commits/e1497a029cf9b4a965c432b8a444e79d09c26eb4))